micrometer-tracing-bridges/micrometer-tracing-bridge-otel/src/main/java/io/micrometer/tracing/otel/bridge/OtelCurrentTraceContext.java

```diff
@@ -129,4 +125,32 @@ public ExecutorService wrap(ExecutorService delegate) {
         return Context.current().wrap(delegate);
     }
 
+    static class WrappedScope implements Scope {
+
+        final io.opentelemetry.context.Scope scope;
+
+        final OtelTraceContext otelTraceContext;
+
+        final Context old;
```

Comment:
What do you think about renaming them to `currentOtelTraceContext` and `oldContext`?